lua/tardis/interiors/base.lua

```diff
@@ -91,6 +91,7 @@ T.Exterior = {
             mat = "drmatt/tardis/mat.wav",
             mat_damaged = "jeredek/tardis/mat_damaged.wav",
             mat_fail = "drmatt/tardis/mat_fail.wav",
+            mat_fast = "Poogie/tardis/mat_fast.wav",
```

Comment:
All files must be lowercase for Linux/macOS compatibility

Note renaming the actual file often won't get picked up you may have to change it to something else entirely, commit, then change it back with lowercase and commit again